server/enterprise/message_export/worker.go

```diff
@@ -22,33 +22,7 @@ import (
 	"github.com/mattermost/mattermost/server/v8/platform/shared/templates"
 )
 
-const (
-	// JobDataBatchStartTimestamp is the posts.updateat value from the previous batch. Posts are selected using
-	// keyset pagination sorted by (posts.updateat, posts.id).
-	JobDataBatchStartTimestamp = "batch_start_timestamp"
-
-	// JobDataStartTimestamp is the start of the job (doesn't change across batches)
-	JobDataStartTimestamp = "job_start_timestamp"
-
-	// JobDataBatchStartId is the posts.id value from the previous batch.
-	JobDataBatchStartId = "batch_start_id"
-
-	// JobDataEndTimestamp is the point up to which this job is exporting. It is the time the job was started,
-	// i.e., we export everything from the end of previous batch to the moment this batch started.
-	JobDataEndTimestamp            = "batch_end_timestamp"
-	JobDataStartId                 = "start_id"
-	JobDataExportType              = "export_type"
-	jobDataBatchSize               = "batch_size"
-	jobDataChannelBatchSize        = "channel_batch_size"
-	jobDataChannelHistoryBatchSize = "channel_history_batch_size"
-	JobDataMessagesExported        = "messages_exported"
-	JobDataWarningCount            = "warning_count"
-	JobDataIsDownloadable          = "is_downloadable"
-	JobDataExportDir               = "export_dir"
-	JobDataBatchNumber             = "job_batch_number"
-	JobDataTotalPostsExpected      = "total_posts_expected"
-	TimeBetweenBatchesMs           = 100
-)
+const TimeBetweenBatchesMs = 100
```

Comment:
Totally unrelated, but could you remind me if we figured out this value through benchmarking or if it preceded your work? It feels like it could be valuable to have a degree of control on this, but maybe it doesn't matter so feel free to disregard.

Reply:
No, it's an old magic number we've always had as a way to break up the batches. I'm on the fence whether we should add a config val or env var for it, considering that in all the testing I've done we've never gone above 2% cpu during any export task 😂 . We're db (for brief moments) and network constrained (during everything else) not cpu bound.

Reply:
I thought the main use of that was to give the DB a "breather" between consecutive batches. Of course, the batch size is the other big variable here. As the batch size becomes smaller I'd expect this value to become more important. Anyhow, making it env controllable would be a good first step in my mind, but we can definitely get back to this later.

Reply:
> I thought the main use of that was to give the DB a "breather" between consecutive batches as the app is mostly blocked on IO anyway.

Maybe that's what it was originally intended for, but in practice that's not a concern -- the db is used only briefly and then we wait on network IO for a long time (comparatively). 
Let's come back to it later, because I don't think it's has any practical effect at all.

Reply:
> Let's come back to it later, because I don't think it's has any practical effect at all.

Fair. If that's the case, though, I'd encourage us to consider removing it altogether. To be decided when, of course.